src/components/HtmlView.js

```diff
@@ -73,23 +69,30 @@ export const HtmlView = memo(({ html, tagsStyles, openWebScreen, width }) => {
 
   return (
     <HTML
-      html={html}
+      source={{ html }}
       {...htmlConfig}
-      onLinkPress={(evt, href) => openLink(href, openWebScreen)}
+      renderersProps={{
+        a: { onPress: (evt, href) => openLink(href, openWebScreen) },
+        table: { cssRules }
+      }}
       tagsStyles={{ ...styles.html, ...tagsStyles }}
       emSize={normalize(16)}
-      baseFontStyle={styles.baseFontStyle}
+      baseStyle={{ ...styles.baseFontStyle }}
       ignoredStyles={['width', 'height']}
-      imagesMaxWidth={maxWidth}
-      staticContentMaxWidth={maxWidth}
-      alterChildren={(node) => {
-        if (node.name === 'img' || node.name === 'iframe') {
-          delete node.attribs.width;
-          delete node.attribs.height;
-        }
+      contentWidth={maxWidth}
+      domVisitors={{
+        onElement: (node) => {
+          // Add an image extracted from data-cover-src
+          // attr as first child of every article.
```

Comment:
what does this comment mean? what is `data-cover-src`?

Reply:
I deleted it.
https://github.com/smart-village-solutions/smart-village-app-app/commit/bcc80d212f5cbe719c85ba9c5c36b2df18629b96